pages/your_tasks/machine_actionability.md

```diff
@@ -40,7 +40,7 @@ More and more often, funders, data managers/stewards, IT staff and institutions
 * While providing an all-encompassing definition for this topic is not within the scope of this platform, it is important to clarify that (meta)data can be used by machines to different extents, depending on its characteristics. Here, we report a few common definitions.
     * "Machine-readable: data in a data format that can be automatically read and processed by a computer, such as [CSV](https://opendatahandbook.org/glossary/en/terms/csv/), [JSON](https://opendatahandbook.org/glossary/en/terms/json/), [XML](https://opendatahandbook.org/glossary/en/terms/xml/), etc. Machine-readable data must be [structured data](https://opendatahandbook.org/glossary/en/terms/structured-data/).", [Open Data Handbook](https://opendatahandbook.org/glossary/it/terms/machine-readable/).
     * "Machine-readable data, or computer-readable data, is data in a format that can be processed by a computer. Machine-readable data must be structured data.", [Wikipedia](https://en.wikipedia.org/wiki/Machine-readable_data).
-    * "Machine-actionable: this term refers to information that is structured in a consistent way so that machines, or computers, can be programmed against the structure.", [DDI](https://ddialliance.org/taxonomy/term/198).
+    * "Machine-actionable data: structured data that are represented in a way so that a machine (computer) can be programmed to read and process each datum [representation of a concept intended for information processing purposes]", [DDI](https://ddialliance.org/glossary).
```

Comment:
@floradanna These square brackets, are they fine?

Reply:
```suggestion
    * "Machine-actionable data: structured data that are represented in a way so that a machine (computer) can be programmed to read and process each datum representation of a concept intended for information processing purposes", [DDI](https://ddialliance.org/glossary).
```